packages/ens-utils/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# dependencies
+/node_modules
```

Comment:
Why are we making this change? This suggests we aren't running our `pnpm` commands from the root of the monorepo. I don't see why we should do this?